src/confd/yang/infix-services.yang

```diff
@@ -36,6 +59,49 @@ module infix-services {
       type boolean;
     }
   }
+  container ssh {
+    description "Configuration for the SSH daemon";
+    leaf enabled {
+      description "Disable or enable SSH daemon";
+      type boolean;
+      must ". = 'false' or (count(../listen) > 0 and count(../hostkey) > 0)" {
+        error-message "When SSH is enabled, there must be at least one listen address configured and a private key must be set.";
+      }
+    }
+
+    leaf-list hostkey {
+      description
+        "A reference to an asymmetric key that exists in
+          the central keystore.";
```

Comment:
For the sanity of people reading help texts.  The first sentence should be complete and without line breaks and kept within 70 chars.

Suggest simplify:

    description "Reference to asymmetric key in central keystore.

                 The hostkey can be shared with NETCONF, by default 'genkey' is used."